tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -3479,6 +3479,46 @@ public function assertNoSharesOfUserAreInState(string $user, string $state):void
 		);
 	}
 
+	/**
+	 * @Given /^user "([^"]*)" has unshared folder "([^"]*)" form "([^"]*)"$/
+	 *
+	 * @param string $sharer
+	 * @param string $path
+	 * @param string $sharee
+	 *
+	 * @return void
+	 * @throws JsonException
+	 */
+	public function userHasUnsharedFolderForm(string $sharer, string $path, string $sharee): void {
```

Comment:
```suggestion
	public function userHasUnsharedEntityFrom(string $sharer, string $path, string $sharee): void {
```
since i see both folder and file unshared